docs/admin/access.rst

```diff
@@ -5,56 +5,56 @@ Access control
 
 .. versionchanged:: 3.0
 
-    Before Weblate 3.0, the privileges system was based on Django, but now it
-    is built specifically for Weblate. If you are using older version, please
-    consult documentation for that version, information here will not apply.
+    Before Weblate 3.0, the privilege system was based on Django, but is now
+    specifically built for Weblate. If you are using an older version, please
+    the consult documentation for that version, the information here will not apply.
 
-Weblate comes with a fine grained privileges system to assign users permissions
-globally or in limited scope.
+Weblate comes with a fine grained privilege system to assign user permissions
+for the whole instance, or in a limited scope.
```

Comment:
Not quite sure what all of the "scope" relates to.

Reply:
Probably privileges per project, component or language.